zeebe/gateway-rest/src/test/java/io/camunda/zeebe/gateway/rest/controller/usermanagement/GroupQueryControllerTest.java

```diff
@@ -193,18 +226,134 @@ void shouldSortAndPaginateSearchResult() {
             """
             {
               "sort":  [{"field": "name", "order":  "asc"}],
-              "page":  {"from":  20, "limit":  10}
+              "page":  {"from":  20, "limit":  2}
             }
-             """)
+            """)
         .exchange()
         .expectStatus()
-        .isOk();
+        .isOk()
+        .expectBody()
+        .json(EXPECTED_SEARCH_RESPONSE);
 
     verify(groupServices)
         .search(
             new GroupQuery.Builder()
                 .sort(GroupSort.of(builder -> builder.name().asc()))
-                .page(SearchQueryPage.of(builder -> builder.from(20).size(10)))
+                .page(SearchQueryPage.of(builder -> builder.from(20).size(2)))
                 .build());
   }
+
+  @Test
+  void shouldFailAnInvalidSearchQuery() {}
+
+  @ParameterizedTest
+  @MethodSource("invalidGroupSearchQueries")
```

Comment:
👍 I like the parameterization!

Reply:
I can't take credit for it, it was already very well done in the `AuthorizationQueryControllerTest` 🙂